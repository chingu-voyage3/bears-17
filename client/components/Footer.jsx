/* eslint-disable jsx-a11y/accessible-emoji */
import React from 'react';

import Emoji from 'Components/Emoji';

const Footer = () => (
  <footer className="site-footer">
    <div className="wrapper">
      <p>Made with <Emoji label="Love">❤️</Emoji> by the Chingu Bears 17 team</p>
      <p>Source code is available on our <a href="https://github.com/chingu-voyage3/bears-17" target="_blank" rel="noopener noreferrer">GitHub page</a>
      </p>
    </div>
  </footer>
);

export default Footer;
