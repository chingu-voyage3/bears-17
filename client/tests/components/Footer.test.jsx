import React from 'react';
import renderer from 'react-test-renderer';

import Footer from 'Components/Footer';

describe('Footer', () => {
  it('should match the snapshot', () => {
    const footer = renderer.create(<Footer />);

    expect(footer.toJSON()).toMatchSnapshot();
  });
});
