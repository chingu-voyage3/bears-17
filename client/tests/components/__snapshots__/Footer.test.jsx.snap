// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
<footer
  className="site-footer"
>
  <div
    className="wrapper"
  >
    <p>
      Made with 
      <span
        aria-label="Love"
        className="emoji"
        role="img"
      >
        ❤️
      </span>
       by the Chingu Bears 17 team
    </p>
    <p>
      Source code is available on our 
      <a
        href="https://github.com/chingu-voyage3/bears-17"
        rel="noopener noreferrer"
        target="_blank"
      >
        GitHub page
      </a>
    </p>
  </div>
</footer>
`;
